src/NServiceBus.AcceptanceTests/Timeout/CyclingOutageTimeoutPersister.cs

```diff
@@ -95,7 +95,7 @@ public Task<TimeoutData> Peek(string timeoutId, ContextBag context)
             return Task.FromResult<TimeoutData>(null);
         }
 
-        public Task<TimeoutsChunk> GetNextChunk(DateTime startSlice)
+        public Task<TimeoutsChunk> GetNextChunk(DateTime startSlice, int maxChunkSize = Int32.MaxValue, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
https://github.com/Particular/NServiceBus/pull/3497/files#r53817062 good point. Like that